tests/spec/spec/prometheus/prometheus_spec.rb

```diff
@@ -110,10 +110,10 @@
     end
 end
 
-describe 'Check connection to HAProxy Exporter hosts' do
-    listInventoryHosts("haproxy_exporter").each do |val|
+describe 'Check connection HAproxy metrics endpoint' do
+    listInventoryHosts("haproxy").each do |val|
         let(:disable_sudo) { false }
-        describe command("curl -o /dev/null -s -w '%{http_code}' $(grep -oP \"(?<=targets: \\\[\').*(?=\'\\\])\" /etc/prometheus/file_sd/haproxy-exporter-#{val}.yml)/metrics") do
+        describe command("curl -o /dev/null -s -w '%{http_code}' #{haproxy_host}:#{haproxy_port}/metrics") do
```

Comment:
`haproxy_host` and `haproxy_port` variables are not defined.
Note that these tests are run on `prometheus` host and `haproxy` host for each cluster may be different.

Reply:
yes, Im aware - that's why its draft